signs-work-order-map/utils/mapUtils.tsx

```diff
@@ -12,10 +12,17 @@ import { Sign, KnackToIFrameMessage, LatLon } from "@/types/map";
  */
 export const useFormatBounds = (signs: Sign[]): undefined | LngLatBoundsLike =>
   useMemo(() => {
-    if (signs.length === 0) {
+    // Knack will save undefined latitudes and longitudes, this filters those out.
+    const checkedSigns = signs.filter((sign: Sign) => sign.lat && sign.lng);
```

Comment:
The interface definition for `Sign` still indicates that `lat` and `lng` will never be undefined. Would you mind updating that?